iroh/src/rpc_protocol.rs

```diff
@@ -55,14 +76,25 @@ impl ServerStreamingMsg<ProviderService> for ListBlobsRequest {
     type Response = ListBlobsResponse;
 }
 
+/// List all collections
+///
+///
```

Comment:
...?  so much suspense!

Reply:
I hate this call, that is why. Not sure what to write here that makes sense since the whole call does not make that much sense.